zellij-server/src/lib.rs

```diff
@@ -277,7 +278,8 @@ pub fn start_server(mut os_input: Box<dyn ServerOsApi>, socket_path: PathBuf) {
                                             client_id,
                                         )
```

Comment:
Sorry, I mixed up the braces here. The call to `.fatal()` goes here, the `.unwrap()` below can stay.